python/python/bystro/ancestry/train.py

```diff
@@ -280,8 +406,8 @@ def _filter_variants_for_fst(genotypes: pd.DataFrame, labels: pd.DataFrame) -> p
 def _load_dataset() -> tuple[pd.DataFrame, pd.DataFrame]:
     """Return the final dataset consisting of genotype matrix, labels."""
     genotypes = _load_genotypes()
-    labels = _load_label_data(genotypes.index)
-    genotypes, labels = _filter_samples_for_relatedness(genotypes, labels)
+    labels = load_label_data(genotypes.index)
+    genotypes, labels = filter_samples_for_relatedness(genotypes, labels)
```

Comment:
Are we filtering samples for relatedness with the idea that a future reference dataset will include related individuals? If not, do we still need this step?

Reply:
we'll probably never want to include related individuals in an ancestry model, so maybe we should canonicalize the sample set and push this step further upstream.  Added TODO to this effect.